delivery_package_fee/tests/test_package_fee.py

```diff
@@ -10,10 +11,15 @@ class TestPackageFee(TransactionCase):
     def setUpClass(cls):
         super().setUpClass()
         cls.product1 = cls.env["product.product"].create(
-            {"name": "Product 1", "type": "product", "lst_price": 1.0}
+            {"name": "Product 1", "is_storable": True, "lst_price": 1.0}
         )
         cls.product2 = cls.env["product.product"].create(
-            {"name": "Product 2", "type": "product", "lst_price": 1.0}
+            {"name": "Product 2", "is_storable": True, "lst_price": 1.0}
+        )
+        cls.pricelist = cls.env["product.pricelist"].create(
+            {
+                "name": "Test Pricelist",
+            }
```

Comment:
why?

Reply:
in `test_package_fee_pricelist`, we retrieve the [pricelist](https://github.com/chaule97/delivery-carrier/blob/1b2e553e2ef943dbb5d848a1a0d90f5d0501c39d/delivery_package_fee/tests/test_package_fee.py#L329) from `sale.order` . The pricelist in `sale.order` is from [order.partner_id.property_product_pricelist](https://github.com/odoo/odoo/blob/a152910e3a352516ea1240d8833a406b144382dd/addons/sale/models/sale_order.py#L438). which is from [_get_partner_pricelist_multi](https://github.com/odoo/odoo/blob/a152910e3a352516ea1240d8833a406b144382dd/addons/product/models/res_partner.py#L34C45-L34C73).

in [_get_partner_pricelist_multi](https://github.com/odoo/odoo/blob/a152910e3a352516ea1240d8833a406b144382dd/addons/product/models/product_pricelist.py#L284), my understanding is that it selects the first active pricelist  (`active=True`) with the same `company_id`. Currently, in the test environment, there are no records in the pricelist table, so I need to create one.

Reply:
ok, can you update the description as well

From Odoo 17.0, Public pricelist was removed in https://github.com/odoo/odoo/pull/84920/commits/40f0482b680a7872d338fa8fd10eec4663637b46, adding pricelist for testing